packages/demo/src/App.svelte

```diff
@@ -239,8 +235,8 @@
     // },
     appMetadata: {
       name: 'Blocknative',
-      icon: blocknativeIcon,
-      logo: blocknativeLogo,
+      // icon: blocknativeIcon,
```

Comment:
@gesquinca what are we thinking here for the demo icons? Remove?

Reply:
yeah, remove so we can test the default design. It's nice to be able to uncomment these for testing custom icon/logo options. That's why I left them there.

Reply:
I leave commented out for boilerplate examples